src/main/java/dev/thource/runelite/dudewheresmystuff/StorageManagerManager.java

```diff
@@ -181,4 +238,36 @@ public void setItemSortMode(ItemSortMode itemSortMode) {
   public void onMenuOptionClicked(MenuOptionClicked menuOption) {
     storageManagers.forEach(manager -> manager.onMenuOptionClicked(menuOption));
   }
+
+  // Creates a CSV file containing all the items in any exportable storage
+  public void exportItems() {
+    if (displayName.equals("")) {
+      log.info("Can't export: no display name");
+      return;
+    }
+    File user_dir = new File(EXPORT_DIR, displayName);
+    String fileName = new SimpleDateFormat("'dudewheresmystuff-'yyyyMMdd'T'HHmmss'.csv'").format(new Date());
+    String filePath = user_dir + File.separator + fileName;
+
+    Collection<ItemStack> items = getWithdrawableItems();
+
+    try
+    {
+      user_dir.mkdirs();
+
+      BufferedWriter writer = new BufferedWriter(new FileWriter(filePath));
+      // Include a CSV header describing the columns
+      writer.write("ID,Name,Quantity\n");
+
+      for (ItemStack stack : items) {
+        // TODO: Figure out how to escape commas or other special characters in item names
+        writer.write(String.format("%d,%s,%d\n", stack.getCanonicalId(), stack.getName(), stack.getQuantity()));
```

Comment:
From what I can understand, only commas need to be replaced: `stack.getName().replace(",", "")`

Reply:
Seems reasonable, I changed it to replace , and \n with empty string. I think some CSV implementations do more complicated things like quoting, but items with commas in them should be rare enough that it shouldn't matter.